src/Microsoft.AspNet.OData.Shared/DefaultEdmPatchMethodHandler.cs

```diff
@@ -0,0 +1,170 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="DefaultEdmODataAPIHandler.cs" company=".NET Foundation">
```

Comment:
The name of the file here doesn't correspond with the actual name of the file

Reply:
Fixed